client/sql/sql_test.go

```
@@ -42,3 +48,299 @@
+}
+
+func (m *mockRows) Columns() []string {
+	return []string{"id", "name"}
+}
+
+func (m *mockRows) Close() error {
+	m.closed = true
+	return nil
+}
+
+func (m *mockRows) Next(dest []driver.Value) error {
```

golangci-lint (2.5.0)

[warning] unused-parameter: parameter 'dest' seems to be unused, consider removing or renaming it as _

(revive)